controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/pricing/PricingApiHandler.java

```diff
@@ -156,6 +156,7 @@ private static SlimeJsonResponse response(PriceInformation priceInfo) {
 
         var array = cursor.setArray("priceInfo");
         addItem(array, "List price", priceInfo.listPrice());
+        addItem(array, "Support level", priceInfo.supportLevelCost());
```

Comment:
Consider changing description to `${supportLevel} support`, e.g. `Enterprise support`.

Reply:
Done